frontend/packages/dev-console/src/components/pipeline-resources/list-page/PipelineResourcesListPage.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { getBadgeFromType } from '@console/shared';
+import { ListPage } from '@console/internal/components/factory';
+import { referenceForModel } from '@console/internal/module/k8s';
+import PipelineResourcesList from './PipelineResourcesList';
+import { PipelineResourceModel } from '../../../models';
+
+interface PipelineResourcesListPageProps {
+  showBadge?: boolean;
+}
+
+const PipelineResourcesListPage: React.FC<Omit<
+  React.ComponentProps<typeof ListPage> & PipelineResourcesListPageProps,
+  'canCreate' | 'kind' | 'ListComponent' | 'rowFilters'
+>> = (props) => {
+  const { showBadge = true } = props;
+  return (
+    <ListPage
+      {...props}
+      canCreate={false}
+      kind={referenceForModel(PipelineResourceModel)}
+      ListComponent={PipelineResourcesList}
```

Comment:
Not sure if you're implementing it fully in this PR or not... but you need list filters for the type column.

Reply:
Yeah was planning to add the filters in the upcoming PR as didnt wanted to stretch the story/PR any more :stuck_out_tongue: